templates/local_storage_operator_localvolume.yaml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "local-disks"
+  namespace: "local-storage" 
+spec:
+  nodeSelector: 
+    nodeSelectorTerms:
+    - matchExpressions:
+      - key: kubernetes.io/hostname
+        operator: In
+        values:
+        {% for host in ocp-ci %}
```

Comment:
If there is no way to automatically have that 'valid' for all nodes in a cluster (so when a master/worker would join it) ? 
If not, that means that each time we add a node, it would need to parse the inventory (which is not a problem per SE), but in that case, the group name used should also be coming from inventory/vars  (with something defined in defaults/main.yml). 
This way, the role would be generic and not using hard-coded group name

Reply:
I'll have to check if thats possible automatically @arrfab, if not.. it could make for a good feature suggestion on the Operator itself. Maybe target node labels or something.